engine/runtime-parser/src/main/java/org/enso/compiler/core/TreeToIr.java

```diff
@@ -1985,8 +1985,6 @@ private IdentifiedLocation getIdentifiedLocation(ArgumentDefinition ast) {
     long end;
     if (ast.getClose() != null) {
       end = ast.getClose().getEndCode();
-    } else if (ast.getDefault() != null) {
```

Comment:
This means that in
```
foo a=1 =
    ...
```
`a`'s location **does not** include `=`.

Reply:
With this change, the logic doesn't seem internally consistent. However, I don't think what it was doing before this change makes sense.

What is the correct `IdentifiedLocation` for a `DefinitionArgument`? I would think it should include the whole argument definition (which is different from the logic before or after this change). But maybe it's only expected to include the variable binding?

If the `IdentifiedLocation` should only include the binding, we should get rid of all these branches and just do `end = ast.getPattern().getEndCode();`

Reply:
IDE definitely doesn't assign UUID to the whole `a=1` when it attaches the visualization. It only attaches it to the parameter. I guess that's expected. 

> If the IdentifiedLocation should only include the binding, we don't need this function, we can just use getIdentifiedLocation(def.getPattern()).

I don't know that code base nor when the different conditionals were supposed to cover for the argument. So if that works, then I'm happy to change it.

Reply:
First, `a=1` in function signature is not a separate AST node, but part of FunctionDef. Therefore, I think we cannot assign UUID to it.

Reply:
Yeah, we should use the location of the parameter. `getIdentifiedLocation(def.getPattern())` instead of this whole function.

Reply:
done